src/NgLamVN/InvCraft/Provider.php

```diff
@@ -35,6 +39,15 @@ public function open(){
 			$this->msg = new Config($this->getLoader()->getDataFolder() . "message.yml", Config::YAML);
 		}
 	}
+	public function stringToItem(string $input): ?Item {

```

Comment:
This function should be static and add to a separate class like "Utils" because it can be use every where in this plugin...

Reply:
Now you can do changes what you want 😜